contrib/nydusify/cmd/nydusify.go

```diff
@@ -528,3 +539,19 @@ func main() {
 		logrus.Fatal(err)
 	}
 }
+
+func setupLogLevel(c *cli.Context) {
+	lvl := c.String("log-level")
+	logLevel, err := logrus.ParseLevel(lvl)
+	if err != nil {
+		logrus.Warnf("failed to parse log level(%s): %+v\ndefault log level(info) will be used", lvl, err)
+		logLevel = defaultLogLevel
+	}
+
+	logrus.SetLevel(logLevel)
+
+	// global `-D` has the highest priority
+	if c.Bool("D") {
```

Comment:
We can check this at the beginning of the func.

Reply:
Check it last to let it overwrite the `--log-level`.

Reply:
How about this:

```
// global `-D` has the highest priority
if c.Bool("D") {
	return logrus.SetLevel(logrus.DebugLevel)
}

lvl := c.String("log-level")
logLevel, err := logrus.ParseLevel(lvl)
if err != nil {
	logrus.Warnf("failed to parse log level(%s): %+v\ndefault log level(info) will be used", lvl, err)
	logLevel = defaultLogLevel
}
logrus.SetLevel(logLevel)
```

Reply:
Or which should have the higher priority?